readme.md

```diff
@@ -158,6 +158,28 @@ You can use the [Angular.js binding](https://github.com/hrajchert/angular-screen
 </div>
 ```
 
+#### Toggle Fullscren as an Angular 2 Directive
+
+```typescript
+import { Directive, HostListener } from '@angular/core';
+const screenfull = require('screenfull/dist/screenfull');
+
+@Directive({
+    selector: '[toggleScreenfull]'
+})
+export class ToggleScreenfullDirective {
+    @HostListener('click') onClick() {
```

Comment:
I believe that's how the Angular Style Guide use attribute decorators: https://angular.io/styleguide#!#06-01